### PR TITLE
fix(network): fix nil pointer dereference when dns name servers are n…

### PR DESCRIPTION
**Pull request by SerseusWasTaken**
…ot set during update

## Description
Fixes a nil pointer dereference when no dns servers are configured for the update.

## Checklist

- [ ] Issue was linked above
- [ ] Code format was applied: `make fmt`
- [ ] Examples were added / adjusted (see e.g. [here](https://github.com/stackitcloud/stackit-cli/blob/ef291d1683ca5b0d719ec0a26ecb999a32685117/internal/cmd/ske/cluster/create/create.go#L49-L63))
- [x] Docs are up-to-date: `make generate-docs` (will be checked by CI)
- [ ] Unit tests got implemented or updated
- [x] Unit tests are passing: `make test` (will be checked by CI)
- [x] No linter issues: `make lint` (will be checked by CI)

### CODE REVIEW

File: internal/cmd/network/update/update.go [hunk 1]
```diff
@@ -192,8 +194,10 @@ func buildRequest(ctx context.Context, model *inputModel, apiClient *iaas.APICli
 	}
 
 	if model.IPv4DnsNameServers != nil || model.NoIPv4Gateway || model.IPv4Gateway != nil {
-		payloadIPv4 = &iaas.UpdateNetworkIPv4Body{
-			Nameservers: *model.IPv4DnsNameServers,
+		payloadIPv4 = &iaas.UpdateNetworkIPv4Body{}
+
+		if model.IPv4DnsNameServers != nil {
+			payloadIPv4.Nameservers = *model.IPv4DnsNameServers
```
Comment (marceljk) [inline internal/cmd/network/update/update.go:200]: We can add here an additional unit test, to cover this issue with the nil pointer dereference, to avoid that this happens again in the future

**Review by marceljk — APPROVED** [review]
